Configuration/TsConfig/Page/newContentElement.tsconfig

```diff
@@ -0,0 +1,27 @@
+mod.wizards.newContentElement.wizardItems {
+  // add the plugins to the "Create new content element" wizard
+  plugins {
+    elements {
+      tea_teaindex {
+        title = LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_index
+        description = LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_index.description
```

Comment:
We also need to add both descriptions to the EN and DE language files.

Reply:
Done